frontend/packages/console-app/src/components/nodes/node-dashboard/queries.ts

```diff
@@ -85,40 +85,16 @@ const top25Queries = {
 
 const resourceQuotaQueries = {
   [NodeQueries.POD_RESOURCE_LIMIT_CPU]: _.template(
-    `sum(
-      max by (namespace, pod, container) (
-          kube_pod_container_resource_limits_cpu_cores{node='<%= node %>', job="kube-state-metrics"}
-      ) * on(namespace, pod) group_left() max by (namespace, pod) (
-          kube_pod_status_phase{phase=~"Pending|Running"} == 1
-      )
-    )`,
+    `sum(kube_pod_resource_limit{node='<%= node %>',resource='cpu'})`,
```

Comment:
nit: You shouldn't need backticks now since the strings no longer span lines and there's no variable replacement. (The `<%= node %>` variable is handled by lodash `_.template`.)

```suggestion
    "sum(kube_pod_resource_limit{node='<%= node %>',resource='cpu'})",
```

pod.tsx should still have the backticks, though.

Reply:
Oops

Reply:
Won't that make this line inconsistent with all other lines in the file?  I.e. do you want these two lines to be inconsistent with all the others?

Reply:
Ah, I see we use template literals everywhere in the file. Don't worry about it then. Sorry.

(I thought we have a eslint rule for unnecessary template literals, but I guess not.)